agent/acs/model/api/api-2.json

```diff
@@ -437,7 +437,7 @@
     },
     "SecretProvider":{
       "type":"string",
-      "enum":["SSM"]
+      "enum":["ssm"]
```

Comment:
How come this line changed? The ACS model is changed since your last PR?

Reply:
im confused. this change should was already merged to `aws:ecs-secrets` here https://github.com/aws/amazon-ecs-agent/commit/ebc8c943ec9b72f1a31e91845f53c82bb098ab01

what am i missing?

Reply:
So seems the sha changes, so pr takes it as a new change. However, there is no more changes besides what included in ebc8c94 actually.